python/lsst/ts/atmcssimulator/mcs_csc.py

```diff
@@ -820,10 +820,11 @@ async def update_events(self):
             m3_state = None
             if m3_in_position:
                 # we are either at a port or at an unknown position
-                # exit port enum values = m3state enum values
-                # for the known exit ports
+                # exit port. The m3_state and exit port may have different enum
+                # values but will have the same name. So we need to get the
+                # enum value of M3State that matches the name of the exit port.
```

Comment:
Do we have to live with this? Seems saner to make sure they have the same values.

Reply:
Unfortunately, at least for the time being, we will have to live with this. Mainly because this is what the LabView code is doing... I guess we could update the xml and idl to make the values match, but that means we will need to wait for the next cycle to update it...

I thought about a better solution though... maybe instead of using getattr I can create a dictionary to map the values between both. I think it would probably cleanup a bit of this part of the code. 

Maybe in the future, when the ATMCS switch to tcp/ip we can improve this.

Reply:
If you are willing, please:
* File a ticket to change the ts_xml and ts_idl to match the low-level controller.
* Add a dictionary to convert conversion method for now, with a TO DO comment to get rid of it when the new ticket is implemented.